src/components/availability-label/availability-label.test.js

```diff
@@ -0,0 +1,60 @@
+describe("Availability Label", () => {
+  it("shows that at least one of the materials is available", () => {
+    cy.intercept("GET", "**/external/agencyid/catalog/availability/v3?**", {
```

Comment:
```suggestion
    cy.intercept("GET", "**/external/agencyid/catalog/availability/v3?recordid=62523611&recordid=62150041&recordid=61435867", {
```

Consider including the query parameters in the mocked request.

Since we have unusual requirements regarding the formatting of these arguments in the query string it may be worthwhile to also require that specific formatting in the test.